### PR TITLE
feat: P4ADEV-1448 develop mock C030

### PR DESCRIPTION
**Pull request by RiccardoGiuliani**
#### How Has This Been Tested?

- Pre-Deploy Test
  - [ ] Unit
  - [ ] Integration (Narrow)
- Post-Deploy Test
  - [ ] Isolated Microservice
  - [ ] Broader Integration
  - [ ] Acceptance
  - [ ] Performance & Load

#### Types of changes

- [ ] PATCH - Bug fix (backwards compatible bug fixes)
- [ ] MINOR - New feature (add functionality in a backwards compatible manner)
- [ ] MAJOR - Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] CHORE - Minor Change (fix or feature that don't impact the functionality e.g. Documentation or lint configuration)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Review by antocalo — CHANGES_REQUESTED** [review]
p4pa-mocks it's a generic mocks service, so let's packaging all anpr stuffs into an anpr package

**Review by antocalo — CHANGES_REQUESTED** [review]
SwaggerConfig should be generic not only for anpr

File: src/main/java/it/gov/pagopa/payhub/mocks/anpr/service/AnprMockServiceImpl.java [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+      .build();
+
+    return ResponseEntity.ok(successResponse);
+  }
+
+  private ResponseEntity<RispostaE002OK> processC003Operation(RichiestaE002 request) {
+    throw new IllegalArgumentException("Method not implemented for operation code " + request.getDatiRichiesta().getCasoUso());
+  }
+
+  private String generateIdFromCF(String fiscalCode) {
```
Comment (antocalo) [inline src/main/java/it/gov/pagopa/payhub/mocks/anpr/service/AnprMockServiceImpl.java]: change name method into generateIdAnprFromCF, generateId it's too generics

**Review by antocalo — DISMISSED** [review]

File: src/main/java/it/gov/pagopa/payhub/mocks/anpr/service/AnprMockServiceImpl.java (hunk 1 above)
Comment (RiccardoGiuliani) [reply src/main/java/it/gov/pagopa/payhub/mocks/anpr/service/AnprMockServiceImpl.java]: modified

**Review by antocalo — APPROVED** [review]

**Pull request merged**